### DISCUSSION TITLE
Debug from VSCode?

### DISCUSSION THREAD

**Discussion by FredrikNoren** [DioxusLabs/dioxus#1842] [Q&A]
Hi,

Does anyone know if it's possible to debug dioxus web apps from within VSCode? And if so, what are the steps to make that happen?

**Comment by velocifer-in** [comment]
+1

**Comment by twinforces** [comment]
+1

**Comment by ealmloff** [comment]
I don't have a setup to debug wasm web applications in VSCode, but you can get some browser debugging functionality with a chrome extension as described [here](https://developer.chrome.com/docs/devtools/wasm)

**Reply by twinforces** [reply]
Cool, can you explain like I'm an idiot how to setup dx to do this? I'm not seeing anything in the Chrome debugger for it.

**Comment by Craig-Macomber** [comment]
There is an issue tracking lack of documentation in this area. I have included some debugging approaches in that thread which provide some options that may be useful https://github.com/DioxusLabs/dioxus/issues/3796#issuecomment-2859897531 . If you are interested in debugging, I suggest subscribing to that issue so you will get an update if any official documentation is added to address it.
